### DISCUSSION TITLE
background-color on smartphone (iPhone 13)

### DISCUSSION THREAD

**Discussion by olewaldo** [PrivateBin/PrivateBin#1123] [Q&A] [answered]
Hi there,

can someone tell me, where to change the background color of the body? On my iPhone 13 there is the following color at the top of my screen: `#fbe37c` 

I tried everything but nothing worked. Where is it defined?

Thank you so much!

**Comment by ppfeufer** [comment, marked as answer]
It's more like `#ffe57e` and it's defined as the theme colour here:

![image](https://github.com/PrivateBin/PrivateBin/assets/2989985/58606bc9-1b10-49e6-82b8-501dc2034015)

Which is used by mobile browsers to colour the browser tabs.

**Reply by olewaldo** [reply]
thats it - thank you so much! :)
